src/main/java/io/stargate/sgv2/jsonapi/service/bridge/config/DocumentConfig.java

```diff
@@ -26,18 +26,21 @@
 @ConfigMapping(prefix = "stargate.document")
 public interface DocumentConfig {
 
-  /** @return Defines the maximum document page size, defaults to <code>500</code>. */
-  @Max(500)
-  @Positive
-  @WithDefault("100")
-  int maxPageSize();
-
   /** @return Defines the default document page size, defaults to <code>20</code>. */
   @Max(500)
   @Positive
   @WithDefault("20")
   int defaultPageSize();
 
+  /**
+   * @return Defines the default document page size for sorting, having separate config because sort
+   *     will more rows in per page, defaults to <code>100</code>.
+   */
+  @Max(500)
+  @Positive
+  @WithDefault("100")
+  int defaultSortPageSize();
+
```

Comment:
Let's create a ticket for documenting all config options?

Reply:
Yes will do it.